<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewighet</class>
 <widget class="QWidget" name="gamewighet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>908</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>120</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="wihte">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>白棋</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_white">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/x.jpg);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="whitepos">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>350</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="black">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>黑棋</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_black">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/x.jpg);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="blackpos">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lcd_black</zorder>
   <zorder>black</zorder>
   <zorder>blackpos</zorder>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>810</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>220</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>600</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>670</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>保存棋局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>740</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加载棋局</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
